examples/platformer/systems.rs

```diff
@@ -67,7 +67,7 @@ pub fn movement(
             velocity.linear.y = (up - down) * 200.;
         }
 
-        if input.just_pressed(KeyCode::Space) {
+        if input.just_pressed(KeyCode::Space) && ground_detection.on_ground {
```

Comment:
Let's make it so you can jump out of climbing:
```suggestion
        if input.just_pressed(KeyCode::Space) && (ground_detection.on_ground || climber.climbing) {
```